fdbserver/workloads/FuzzApiCorrectness.actor.cpp

```diff
@@ -177,7 +179,7 @@ struct FuzzApiCorrectnessWorkload : TestWorkload {
 		minNode = std::max(minNode, nodes - newNodes);
 		nodes = newNodes;
 
-		if (useSystemKeys && deterministicRandom()->coinflip()) {
+		if (useSystemKeys) {
```

Comment:
I think we probably still want to have the option of using a non-system prefix with the system keys option, but in that case we should properly detect errors if the tenant mode is required.